### ISSUE TITLE
Replace estimated earnings with Brave Ads views in Rewards 3.0

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#42208]
Replace with:

> You're earning for seeing
> 
> X Ads
> 
> this month

**Comment by GeetaSarvadnya** [comment]
Verification **INPROGRESS** on

```
Brave | 1.75.139 Chromium: 132.0.6834.57 (Official Build) nightly (64-bit)
-- | --
Revision | 824a4385d099e741400ccd850d3309105a1f2c5e
OS | Windows 10 Version 22H2 (Build 19045.5247)
```

**Comment by GeetaSarvadnya** [comment]
@Miyayes : On rewards non-connected state, the estimated earning message displayed as `You could be getting paid for the
1 Ad you've seen this month.` (after viewing an Ad) and `You've seen 0 Ads this month`  (before viewing an Ad).  These messages are NOT replaced by `You're earning for seeing X ads this month` - The updated message is applicable for connected state? or should it apply for non-connected state also?

cc: @Miyayes @zenparsing @LaurenWags @kleantzogu 

**Message before viewing an Ad:**

Rewards settings page | BR panel
--------------------------|----------
![Image](https://github.com/user-attachments/assets/4e89910e-2c40-443b-b8a1-26bab0100c5e) | ![Image](https://github.com/user-attachments/assets/544841c8-5275-4cfa-964c-339cbcf6bae1)

**Message after viewing an Ad:**

Rewards settings page | BR panel
--------------------------|----------
![Image](https://github.com/user-attachments/assets/bbe8f29e-12f6-4a69-b1ed-53c7c92a166d) | ![Image](https://github.com/user-attachments/assets/5048b9f1-9197-418e-9dac-37bad8f72103)

**Comment by Miyayes** [comment]
@GeetaSarvadnya Yes, the text around the number of Ads seen in the non-connected state is different than the text in the connected state. Thanks for noticing, but it is expected. Screenshots look correct :)
